docs/DeveloperGuide.md

```diff
@@ -251,77 +270,377 @@ _{Explain here how the data archiving feature will be implemented}_
 
 --------------------------------------------------------------------------------------------------------------------
 
-## **Appendix: Requirements**
+## **Appendix: Requirements** <a name="requirements"></a>
 
-### Product scope
+### Product scope <a name="scope"></a>
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
+* avid social media user
+* prefers desktop apps over other types
 * prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
+* needs to manage a significant number of contacts
+* wants to catch up with their contacts’ activities quickly 
+* wants to connect with their followers/friends on various social media platforms through an all-in-one dashboard
 
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+**Value proposition**: 
 
+Our product serves as an integrated dashboard for a user to retrieve the social media activities and account information of his / her contacts. This makes it seamless for the user to interact with his / her contacts instead of having to access each social media account that the contact owns.
 
-### User stories
+### User stories <a name="user-stories"></a>
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
-| Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
-| -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+Categories: CF - Core Functionalities, G - Guide, OOC - Organization of Contacts, EOA - Ease of Accessibility, COC - Customization of Contacts
```

Comment:
Personally I'd consider removing the acronyms and instead write the categories in the table directly so it's less confusing, but this is just a minor nitpick